src/client/NetworkingClient.cpp

```diff
@@ -44,16 +45,16 @@ sf::Uint32 NetworkingClient::Start() {
     return mClientId;
 }
 
-std::vector<position> NetworkingClient::getPositions() {
-    while (!mAcceptingPositionRead) {
-        //
+std::vector<network_game_object> NetworkingClient::getNetworkGameObjects() {
+    if (mAcceptingNetworkGameObjectsRead) {
+        return mNetworkGameObjects;
     }
-    return mPositions;
+    return std::vector<network_game_object> {};
 }
 
-void NetworkingClient::setDirection(direction dir) {
+void NetworkingClient::setDirection(sf::Vector2f direction) {
```

Comment:
why change away from a dedicated struct for direction objects? what if we want to make direction 3d one day? don't really see the benefit here except maybe clarity